slides/developer-guide/working-with-documents/create-new-presentations/use-a-pdf-document/_index.md

```diff
@@ -205,7 +223,8 @@ class Test
 
         using (var pdfStream = File.OpenRead("test.pdf"))
         {
-            var response = api.ImportFromPdf("summary.pptx", pdfStream);
+            PdfImportOptions options = new PdfImportOptions();
+            var response = api.ImportFromPdf("summary.pptx", pdfStream, options);
```

Comment:
The `DetectTables` option has not been aplied here.

Reply:
will be fixed